### DISCUSSION TITLE
Is this a Bug or a normal Feature?

### DISCUSSION THREAD

**Discussion by Wasca** [netbox-community/netbox-docker#488] [Getting Help] [answered]
I'm running netbox in Docker and just noticed that when I add an image to a site it creates a URL like this to view the image (https://netbox.yourdomain.com/media/image-attachments/site_1_imagename_DA3Udxm.png)

This URL is accessible even if you're not logged into netbox, i.e. Anyone with the URL and has access to the netbox install can view the image without logging in. I just want to check if this is intended, I thought you'd need to be logged in to view these images as they could reveal sensitive information.

![image](https://user-images.githubusercontent.com/869112/114109568-de228080-9918-11eb-88bf-d6bc07cce388.png)

**Comment by johnnyplaydrums** [comment, marked as answer]
I think that is a normal feature: https://netbox.readthedocs.io/en/stable/configuration/optional-settings/#login_required

If you want to put _all_ Netbox content behind a login, set `LOGIN_REQUIRED = True` in your config.

**Reply by Wasca** [reply]
Thanks @johnnyplaydrums 

I'm very new to netbox and was unaware of this feature, much appreciated!

**Reply by johnnyplaydrums** [reply]
Ya no problem! I'm new to Netbox too and happened across this topic in the docs earlier today.
